### PR TITLE
Add note to 'Scheduling Backup' section to set backup every 10 minutes

### PR DESCRIPTION
**Pull request by HagayVider1**
GH# : 
OCPBUGS# : [OADP-1367](https://issues.redhat.com//browse/OADP-1367)
OSDOCS# : [OADP-1367](https://issues.redhat.com//browse/OADP-1367)

Added note to 'schedule:' cron for scheduling backups every 10 minutes.

Version(s): OADP 1.2.0

Issue: https://issues.redhat.com/browse/OADP-1367

Version(s):

Enterprise 4.11 → branch/enterprise-4.11
Enterprise 4.12 → branch/enterprise-4.12
Enterprise 4.13 → branch/enterprise-4.13
Enterprise 4.14 → branch/enterprise-4.14
Enterprise 4.15 → branch/enterprise-4.15

Issue: [OADP-1367](https://issues.redhat.com//browse/OADP-1367)

Link to docs preview: 
https://59713--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-scheduling-backups-doc

QE review:
- [X] QE has approved this change.

David Vanunu has approved this.

Additional information:

### CODE REVIEW

**Comment by anarnold97** [comment]
@HagayVider1 please can you add the necessary labels to move this to OCP review
Thanks

**Comment by HagayVider1** [comment]
Added link to doc preview.

File: modules/oadp-scheduling-backups.adoc [hunk 1]
```diff
@@ -59,6 +59,16 @@ spec:
 EOF
 ----
 <1> `cron` expression to schedule the backup, for example, `0 7 * * *` to perform a backup every day at 7:00.
+
+[NOTE]
+====
+To schedule a backup every number of minutes, enter the number of minutes to start the next backup in the following format.
```
Comment (snarayan-redhat) [inline modules/oadp-scheduling-backups.adoc:65]:
```suggestion
To schedule a backup at specific intervals, enter the interval duration in minutes in the following format:
```

File: modules/oadp-scheduling-backups.adoc [hunk 2]
```diff
@@ -59,6 +59,16 @@ spec:
...
+----
+  schedule: "*/10 * * * *"
```
Comment (snarayan-redhat) [inline modules/oadp-scheduling-backups.adoc:67]: can we have a placeholder like `<duration_in_minutes>` instead of adding a separate statement below?

**Review by snarayan-redhat — COMMENTED** [review]
couple of suggestions. Otherwise looks good.

**Comment by kalexand-rh** [comment]
I've opened the `enterprise-4.15` branch. Please review your PR and update the applicable versions to also cover 4.15 if you expect that the changes in this PR will be valid in that branch.

File: modules/oadp-scheduling-backups.adoc (hunk 2 above)
Comment (HagayVider1) [reply modules/oadp-scheduling-backups.adoc:68]: Changed as requested

**Comment by anarnold97** [comment]
@HagayVider1 - Please do not ask for this to be merged

**Comment by abhatt-rh** [comment]
@HagayVider1 Kindly provide an updated preview link, squash your commits, and then re-tag for merge. Removing the merge-needed label for now.

/remove-label merge-review-needed
/remove-label merge-review-in-progress

**Comment by anarnold97** [comment]
https://59713--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-scheduling-backups-doc

**Comment by jab-rh** [comment]
@HagayVider1, please squash your commits. Thanks!

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-scheduling-backups-doc.adoc [hunk 3]
```diff
@@ -61,10 +61,21 @@ spec:
     ttl: 720h0m0s
 EOF
 ----
+
 <1> `cron` expression to schedule the backup, for example, `0 7 * * *` to perform a backup every day at 7:00.
++
+[NOTE]
+====
+To schedule a backup at specific intervals, enter the `<duration in minutes>` in the following format:
```
Comment (adellape) [inline backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-scheduling-backups-doc.adoc:69]: Underscores needed for multi-word replaceable values ([repo guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#command-syntax-for-replaceable-values)):
```suggestion
To schedule a backup at specific intervals, enter the `<duration_in_minutes>` in the following format:
```

**Review by adellape — COMMENTED** [review]
Please address the underscores issue and squash down to 1 commit, then re-queue for merge-review. Thank you!

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-scheduling-backups-doc.adoc [hunk 4]
```diff
@@ -61,10 +61,21 @@ spec:
...
+----
+  `schedule: "*/10 * * * *"`
+----
```
Comment (adellape) [inline backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-scheduling-backups-doc.adoc:72]: Also you don't need the backticks within a code block, which should have `[source,terminal]` as well:
```suggestion
[source,terminal]
----
  schedule: "*/10 * * * *"
----
```

File: backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-scheduling-backups-doc.adoc [hunk 5]
```diff
@@ -61,10 +61,21 @@ spec:
...
+Enter the minutes value between quotation marks(" ").
```
Comment (adellape) [inline backup_and_restore/application_backup_and_restore/backing_up_and_restoring/oadp-scheduling-backups-doc.adoc:73]: Spacing and missing backticks:
```suggestion
Enter the minutes value between quotation marks (`" "`).
```
Although I don't know if the parenthetical is really necessary:
```suggestion
Enter the minutes value between quotation marks.
```
Up to you.

**Review by adellape — COMMENTED** [review]
Sorry couple more things.

**Review by emarcusRH — APPROVED** [review]
/lgtm

**Pull request merged**